This patch adds an offline event to the event list for a message
when converting it from a stanza to a message

Index: iris/xmpp-im/types.cpp
===================================================================
--- iris/xmpp-im/types.cpp	(revision 513088)
+++ iris/xmpp-im/types.cpp	(working copy)
@@ -1722,6 +1722,8 @@
 				d->eventList += ComposingEvent;
 			else if (evtag == "delivered")
 				d->eventList += DeliveredEvent;
+			else if (evtag == "offline")
+				d->eventList += OfflineEvent;
 		}
 		if (d->eventList.isEmpty())
 			d->eventList += CancelEvent;
